joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/passes/ast/AssignCpgTests.scala

```diff
@@ -0,0 +1,176 @@
+package io.joern.rubysrc2cpg.passes.ast
+
+import io.joern.rubysrc2cpg.testfixtures.RubyCode2CpgFixture
+import io.shiftleft.codepropertygraph.generated.{EvaluationStrategies, NodeTypes, DispatchTypes, Operators, nodes}
+import io.shiftleft.semanticcpg.language.*
+import io.shiftleft.semanticcpg.language.types.structure.NamespaceTraversal
+
+class AssignCpgTests extends RubyCode2CpgFixture {
+  "single target assign" should {
+    val cpg = code("""x = 2""".stripMargin)
+
+    // TODO: .code property need to be fixed
+    "test assignment node properties" ignore {
+      val assignCall = cpg.call.methodFullName(Operators.assignment).head
+      assignCall.code shouldBe "x = 2" // "="
```

Comment:
`.code `property is given by the parser. The value in it is as per what the parser gives. The cannot be compared directly to other frontends

Reply:
but this is a low priority